src/Components/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface ButtonsProps {
+  loadAllGoods: () => void;
+  loadFiveFirstGoods: () => void;
+  loadRedGoods: () => void;
+}
+
+export const Buttons = ({ loadAllGoods, loadFiveFirstGoods, loadRedGoods }: ButtonsProps) => (
```

Comment:
`export const Component: FC<Props> = () => {}`

Reply:
not fixed